packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1634,6 +1637,28 @@ class _TabBarState extends State<TabBar> {
       );
     }
 
+    final AlignmentGeometry effectiveAlignment = effectiveTabAlignment == TabAlignment.center
```

Comment:
This should be a switch so the analyzer catches this case if TabAlignment ever changes.
Thanks to Dart 3 though, it can be super fun like:

```dart
final AlignmentGeometry effectiveAlignment = switch (effectiveTabAlignment) {
      TabAlignment.center => Alignment.center,
      TabAlignment.start || TabAlignment.startOffset || TabAlignment.fill => AlignmentDirectional.centerStart,
    };
```

Reply:
This looks cool

Reply:
Right? I like it.